src/goto-cc/compile.cpp

```diff
@@ -542,6 +542,11 @@ bool compilet::write_bin_object_file(
   const std::string &file_name,
   const goto_modelt &src_goto_model)
 {
+  status() << "Validating goto model" << eom;
```

Comment:
I don't think this should be printed unconditionally?!

Reply:
fixed